types/immutability-helper/immutability-helper-tests.ts

```diff
@@ -1,48 +1,37 @@
 import * as update from 'immutability-helper';
 import { newContext } from 'immutability-helper';
 
-namespace TestObjectUpdate {
-  update({}, {
-    foo: {
-      bar: { $set: 'baz' }
-    }
-  });
-}
-
-namespace TestArrayUpdate {
-  update([], {
-    foo: {
-      bar: { $set: 'baz' }
-    }
-  });
-}
-
 namespace TestExtend {
   update.extend('$command', (specValue, originalValue) => originalValue);
 }
 
 namespace TestNewContext {
   update.newContext().extend('$command', (specValue, originalValue) => originalValue);
   newContext().extend('$command', (specValue, originalValue) => originalValue);
-
-  // This shouldn't compile, but we can't test negatives.
-  // newContext().newContext();
 }
 
 namespace TestFromReactDocs {
-  // These are copied from https://facebook.github.io/react/docs/update.html
-  let initialArray = [1, 2, 3];
-  let newArray = update(initialArray, { $push: [4] }); // => [1, 2, 3, 4]
 
-  let collection = [1, 2, { a: [12, 17, 15] }];
-  let newCollection = update(collection, { 2: { a: { $splice: [[1, 1, 13, 14]] } } });
-  // => [1, 2, {a: [12, 13, 14, 15]}]
-
-  let obj = { a: 5, b: 3 };
-  let newObj = update(obj, { b: { $apply: function(x: number) { return x * 2; } } });
-  // => {a: 5, b: 6}
-  let newObj2 = update(obj, { b: { $set: obj.b * 2 } });
+    interface TestFoo {
+        bar: number,
+        baz: string,
+        qux: boolean[],
+    }
 
-  let objShallow = { a: 5, b: 3 };
-  let newObjShallow = update(obj, { $merge: { b: 6, c: 7 } }); // => {a: 5, b: 6, c: 7}
+    // Test array type coherence
+    let initialArray: number[] = [1, 2, 3];
+    initialArray = update(initialArray, {$push: [4]}); // => [1, 2, 3, 4]
+
+    // Test Object type coherence
```

Comment:
Can you make this test file consistently formatted? Either commented blocks or namespaced blocks, but mixing both is noisy.